openedxscorm/scormxblock.py

```diff
@@ -420,14 +383,20 @@ def extract_package(self, package_file):
     def index_page_url(self):
         if not self.package_meta or not self.index_page_path:
             return ""
+        
+        if not self.xblock_settings.get("DEFAULT_STORAGE_URL", False):
```

Comment:
I think we should use S3_QUERY_AUTH setting to determinse whetere we should serve static assets vis lms or not. because this setting is used for signing urls in case of private bucket.

```suggestion
if self.xblock_settings.get("S3_QUERY_AUTH",True)
```

Reply:
I think this is a better approach but this limits the setting to just the S3 storage backend. Since we generalized the proxy assets to use any storage, it would be better to provide a generic option for all default django storage backends.

I do think that we can update the name of the setting to something better